templates/webapps/galaxy/dataset/tabular_chunked.mako

```diff
@@ -9,6 +9,7 @@
     ${parent.javascript_app()}
 
     <script type="text/javascript">
+    %if dataset.datatype.file_ext != 'ipynb':
```

Comment:
Totally fine with the hack - if you wanted to build a framework it'd maybe be worth checking subclasses with https://github.com/galaxyproject/galaxy/pull/9955/files and we could probably do json pretty easily then (https://github.com/galaxyproject/galaxy/issues/10192).

Reply:
Well, I do think we should try to avoid mako here and just redirect to a client-side route ... that's my high level view, but right now I have no idea how to do that. @dannon mentioned he might be up for helping with that.

Reply:
We could maybe do this entirely on the client-side, skipping the `/datasets/<dataset>/id?preview=true` thing (if we know we've got a datatype that doesn't do anything special on the backend).